pw_mgr/lib/crypto_utils.py

```diff
@@ -0,0 +1,63 @@
+from cryptography.fernet import Fernet
+
+
+def write_key():
+    """
+    if key.key exists, load key.key. otherwise, create key.key.
+    """
+    with open ("key.key", "wb") as key_file:
```

Comment:
I added a comment about this in your issue but this path needs to be updated so it doesnt change based off where you are running the file